doc/src/learn/exchange-integration-guide.md

```diff
@@ -0,0 +1,238 @@
+curl -X POST -H 'Content-type: application/json' --data-raw "$data" $rpc
+```
+
+The response is a JSON object that includes the totalBalance for the address:
+```json
+{
+  "jsonrpc":"2.0",
+  "result":{
+     "coinType":"0x2::sui::SUI",
+     "coinObjectCount":40,
+     "totalBalance":10000000000,
+     "lockedBalance":{
+       
+     }
+  },
+  "id":1
+}
+```
+
+The following example demonstrates using sui_getBalance in Rust:
+```rust
+use std::str::FromStr;
+use sui_sdk::types::base_types::SuiAddress;
+use sui_sdk::SuiClient;
+
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+   let sui = SuiClient::new("https://fullnode.devnet.sui.io:443", None, None).await?;
+   let address = SuiAddress::from_str("0xa38bc2aa63c34e37821f7abb34dbbe97b7ab2ea2")?;
+   let objects = sui.read_api().get_balance(address).await?;
+   println!("{:?}", objects);
+   Ok(())
+}
+```
+
+## Use events to track balance changes for an address
+
+You can also track the balance for an address by subscribing to all of the events emitted from it. Use a filter to include only the events related to SUI coins, such as when the address acquires a coin or pays for a gas fee.
+The following example demonstrates how to filter events for an address using bash and cURL:
+
+```bash
+rpc="https://fullnode.devnet.sui.io:443"
+address="0xa38bc2aa63c34e37821f7abb34dbbe97b7ab2ea2"
+data="{\"jsonrpc\": \"2.0\", \"id\":1, \"method\": \"sui_getEvents\", \"params\": [{\"Recipient\": {\"AddressOwner\": \"0xa38bc2aa63c34e37821f7abb34dbbe97b7ab2ea2\"}}, null, null, true ]}"
+curl -X POST -H 'Content-type: application/json' --data-raw "$data" $rpc
+```
+
+The response can include a large number of events. Add pagination to the response using the `nextCursor` key. You can determine the corresponding `txDigest` and `eventSeq` from the id field of a transaction.  
+
+You can add the value instead of the first null inside the params. The second null is an integer that defines how many results (up to 1000) to return and the `true` means ascending order. By using the `nextCursor` the reply can start from a desired point.
+
+The “id” field of any transaction looks like:
+```bash
+"id": {
+         "txDigest": "GZQN9pE3Zr9ZfLzBK1BfVCXtbjx5xKMxPSEKaHDvL3E2",
+         "eventSeq": 6019
+       }
+```
+
+With this data, create a nextCursor as follows:
+```bash
+nextCursor : {"txDigest": "GZQN9pE3Zr9ZfLzBK1BfVCXtbjx5xKMxPSEKaHDvL3E2","eventSeq": 6019}
+```
+
+## Blocks vs Checkpoints
+
+Sui is a DAG-based blockchain and uses checkpoints for node synchronization and global transaction ordering. Checkpoints differ from blocks in the following ways:
+ * Sui creates checkpoints and adds transaction after finality
+ * Checkpoints do not fork, roll back, or reorganize.
+ * Sui creates one checkpoint about every 3 seconds.
+
+### Checkpoint API operations
+
+Sui Checkpoint API operations include:
+ * sui_getCheckpoint - Retrieves the specified checkpoint.
```

Comment:
Shall we link to the json-rpc docs here?

Reply:
Yes, I am still waiting on a few details about staking to finalize this topic. Will add/update links